healthScore/utils.py

```diff
@@ -0,0 +1,87 @@
+from django.utils import timezone
+from datetime import datetime, timedelta
+from django.forms.models import model_to_dict
+import json
+
+from .models import (
+    healthRecord,
+    hospital,
+    hospitalStaff,
+    appointment,
+)
+
+
+def get_health_history_details(request):
```

Comment:
We can have this in views.py itself right?

Reply:
Yeah right. I just want to keep reusable utility functions in a separate file and don't want to make the `views.py` file too long. Should I move it to `views.py`?

Reply:
Oh, that makes sense. We can maybe keep the files like role specific (unless features overlap). 
Like healthCare workers could have like a healthCareViews.py or things like that?

Reply:
Sure, we can do that. Should I name the file `user_utils.py`?

Reply:
Yeah, I think that should be good then. Approving the changes

Reply:
Done, thanks!